modules/jumpstart_ui/lib/scss/components/card.component.scss

```diff
@@ -15,4 +15,14 @@
       }
     }
   }
+
+  .image {
+    img,
+    figure {
+      // Show the media element on the smallest screen sizes.
+      @include grid-media-only('xs') {
```

Comment:
You can actually remove the media query part because we just want it to display all the time regardless of screen width.

Reply:
Doh! Yes, that's a much better idea.